xmodule/tests/test_poll.py

```diff
@@ -24,8 +25,9 @@ class PollBlockTest(unittest.TestCase):
 
     def setUp(self):
         super().setUp()
-        self.system = get_test_system()
-        usage_key = self.system.course_id.make_usage_key(PollBlock.category, 'test_loc')
+        course_key = CourseKey.from_string('/'.join(['org', 'course', 'run']))
```

Comment:
Tiny nit (no need to modify for this PR, but in the future): It's fine to just write out the course key string for this:

```python
CourseKey.from_string('org/course/run')
```

Reply:
Good point. I had to rebase the branch, so I changed this while doing it.